js/utils/dom.js

```diff
@@ -10,6 +10,12 @@ function isScrolledIntoView(el, scrollContainer) {
   return isVisible;
 }
 
+// https://developer.mozilla.org/en-US/docs/Web/API/Node/contains
```

Comment:
I think this might be something Mike added for the scrolling effects? If there's an existing jQuery way of doing it, then it would be good to refactor it at some point.